packages/cli/scaffold/trigger.template.js

```diff
@@ -1,13 +1,13 @@
-// triggers on <%= LOWER_NOUN %> with a certain tag
-const trigger<%= CAMEL %> = (z, bundle) => {
-  const responsePromise = z.request({
+// triggers on a new <%= LOWER_NOUN %> with a certain tag
+const perform = async (z, bundle) => {
```

Comment:
This is the updated version of the template; I still need to spruce up the others. They'll match the improvements in this one though.